pkg/pillar/cmd/volumemgr/volumemgr.go

```diff
@@ -690,9 +694,15 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, ar
 // gcUnusedInitObjects this method will garbage collect all unused resource during init
 func gcUnusedInitObjects(ctx *volumemgrContext) {
 	log.Functionf("gcUnusedInitObjects")
-	gcBlobStatus(ctx)
-	gcVerifyImageConfig(ctx)
-	gcImagesFromCAS(ctx)
+	// TODO: Need to handle GC for kubevirt eve, for now disable it.
+	// There are images and blobs downloaded by kubernetes and eve is not aware of those
+	// We use same containerd repository to store those images.
+	// so for now block GC until we find a proper solution (working on it)
```

Comment:
I thought the plan was to GC the images which EVE-OS has downloaded using its downloader/volumemgr -> containerd path, and defer how we GC any images downloaded directly by containerd. (And for the latter we have other design issues like how we use the datastores, https proxies, multiple uplinks with cost, etc). So I think the design of the other images has more work than GC.

Those other images (k3s, longhorn, etc) tend to be downloaded once and always used AFAIU so GC is not the main issue.

But disabling all GC for EVE-downloaded images means that test machines can run out of disk space (since they use a large range of VM and OCI images as part of different tests.)

Reply:
This is temporarily disabling, before production we have to enable it back. This PR is really to make sure nothing is broken until we have permanent solution. But having said that, I am not clear on why test machines will run out of space, won't they delete the apps after testing ?

Reply:
BTW, just want to be clear, nothing changes here for kvm eve. Its only on kubevirt eve temporarily blocking GC

Reply:
> BTW, just want to be clear, nothing changes here for kvm eve. Its only on kubevirt eve temporarily blocking GC

Understood. But if this potentially causes issues with test machines running out of disk space and we want to be able to run the same tests on kubevirt as we do on kvm, then we risk spending time on debugging this for no reason.